src/test/java/edu/cornell/kfs/fp/batch/service/impl/CreateAccountingDocumentServiceImplTestBase.java

```diff
@@ -235,8 +235,8 @@ protected void assertAccountingDocumentIsCorrect(
         assertTrue("Document was not of the expected type of " + documentClass.getName(), documentClass.isAssignableFrom(actualDocument.getClass()));
         assertEquals("Wrong document number", expectedDocument.getDocumentNumber(), actualDocument.getDocumentNumber());
         
-        assertHeaderIsCorrect((FinancialSystemDocumentHeader) expectedDocument.getDocumentHeader(),
-                (FinancialSystemDocumentHeader) actualDocument.getDocumentHeader());
+        assertHeaderIsCorrect((DocumentHeader) expectedDocument.getDocumentHeader(),
+                (DocumentHeader) actualDocument.getDocumentHeader());

```

Comment:
The DocumentHeader typecast on these two lines is no longer needed.

Reply:
updated